client/src/components/Customization.tsx

```diff
@@ -0,0 +1,97 @@
+import { IoMdArrowBack as Arrow } from "react-icons/io";
+import { useNavigate } from "react-router-dom";
+import { twMerge } from 'tailwind-merge';
+
+import H2 from "./H2";
+import H3 from "./H3";
+import Body from "./Body";
+import Button from "./Button";
+import Toggle from "./Toggle";
+import Slider from "./Slider";
+
+interface CustomizationType extends React.HTMLAttributes<HTMLDivElement> {
+  files: string[];
+}
+
+const Customization = ({
+  files,
+  className,
+  ...rest
+}: CustomizationType) => {
+  const navigate = useNavigate();
+
+  async function generateNotes() {
+    const options = {
+      general: files,
+      llm: null
+    };
+
+    try {
+      const res = await fetch('http://localhost:4567/api/v1/generate', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(options),
+      });
+
+      // status check
+      if (!res.ok) {
+        throw new Error('HTTP error ' + res.status);
+      }
+
+      const data = await res.json();
+      console.log(data);
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  return (
+    <div
+      className={twMerge('flex flex-col items-center gap-10 mx-36', className)}
+      {...rest}
+    >
+      <Button
+        icon={Arrow}
+        variant='tertiary'
+        className='absolute left-0'
+        onClick={() => navigate(-1)}
+      >
+        Back
+      </Button>
+      <div className='flex flex-col text-center gap-5'>
+        <H2>Customize Your Notes</H2>
+        <Body>
+          Toggle the different settings to select which content you want to include
+          in your notes and adjust the verbosity level to control its level of detail!
+        </Body>
+      </div>
+      <div className='flex gap-52'>
+        <div className='flex flex-col gap-3'>
```

Comment:
Later we'll need to read the data from these toggles to send to the server, but I think it's fine to leave it like this for the beta release